index.html

```diff
@@ -329,7 +347,11 @@ <h3>MathML Element Mappings</h3>
 		  </td>
 		  <td class="axapi">
 		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
-		    <span class="subrole">AXSubrole: <code>AXMathUnderOver</code></span>
+		    <span class="subrole">AXSubrole: <code>AXMathUnderOver</code></span><br />
+		    <span class="attrs">AXAttributes:
+		      <code>AXMathBase</code> pointing to a first in-flow child
+		      <code>AXMathOver</code> pointing to a second in-flow child
```

Comment:
I added definitions for other children too, but I guess they are easier to define.